test/util/util.go

```diff
@@ -503,6 +503,26 @@ func ValidateNumManifestworks(ctx context.Context, ocmWorkIts *ocmWorkClient.Cli
 	}, timeout).Should(gomega.Equal(num))
 }
 
+func GetNumDeploymentReplicas(ctx context.Context, wec *kubernetes.Clientset, ns string) int {
+	ginkgo.GinkgoHelper()
+	var replicas int = -1
+	gomega.Eventually(func() error {
+		deployments, err := wec.AppsV1().Deployments(ns).List(ctx, metav1.ListOptions{})
+		if err != nil {
+			return err
+		}
+		if len(deployments.Items) != 1 {
+			replicas = -2
```

Comment:
What is the meaning of replica = -2  or -3 ? It would be helpful to use named constants for this values

Reply:
There is no real meaning. These are just values that can not possibly come from `len(anything)`, so that someone debugging a failure gets a clue.

Reply:
But I have pushed a revision that documents these values.